apps/judicial-system/web/src/routes/Shared/IndictmentOverview/IndictmentOverview.tsx

```diff
@@ -171,20 +172,30 @@ const IndictmentOverview: FC = () => {
           </PageTitle>
           <CourtCaseInfo workingCase={workingCase} />
           {isDefenceUser(user) &&
-            workingCase.defendants?.map((defendant) =>
-              (defendant.subpoenas ?? [])
-                .filter((subpoena) =>
-                  isSuccessfulServiceStatus(subpoena.serviceStatus),
-                )
-                .map((subpoena) => (
-                  <Box key={`${defendant.id}${subpoena.id}`} marginBottom={2}>
-                    <ServiceAnnouncement
-                      defendant={defendant}
-                      subpoena={subpoena}
-                    />
-                  </Box>
-                )),
-            )}
+            workingCase.defendants?.map((defendant) => (
+              <>
```

Comment:
Please add a key here so that we don't get a console warning. 

```suggestion
              <Fragment key={defendant.id}>
```

Note that Fragment is imported from React.